tests/validation/test_validate.py

```diff
@@ -108,3 +109,25 @@ def test_validate_all(self):
 
             with self.assertRaises(STACValidationError):
                 pystac.validation.validate_all(stac_dict, new_cat_href)
+
+    def test_validates_geojson_with_tuple_coordinates(self):
+        """This unit tests guards against a bug where if a geometry
+        dict has tuples instead of lists for the coordinate sequence,
+        which can be produced by shapely, then the geometry still passses
+        validation.
+        """
+        geom = {
+            'type':
+            'Polygon',
+            'coordinates': (((-115.3057626, 36.1265426997), (-115.3057626, 36.1282976997),
+                             (-115.3075176, 36.1282976997), (-115.3075176, 36.1265426997),
+                             (-115.3057626, 36.1265426997)), )
```

Comment:
Style question: I don't know with pystac if you want or don't want the trailing `,`

Reply:
This is actually required, as it ensures tuple creation (or else it's interpreted as parans).

e.g.

```python
> x = (4), print(x)
4
> x = (4,); print(x)
(4,)